src/main/resources/static/javascript/mainApp/groupings.service.js

```diff
@@ -20,20 +20,58 @@
             },
 
             /**
-             * Get page of a grouping.
-             * @param {String} path - The path to the grouping.
+             * @param {String[]} groupPaths - The paths to the grouping.
              * @param {Number} page - The number to fetch.
              * @param {Number} size - The size data chunk to be fetched.
              * @param {String} sortString - String to base sort off of.
              * @param isAscending - On true the data returns in ascending order.
              * @param onSuccess - Function to be called if HTTP request returns as a success.
              * @param onError - Function to be called if HTTP request returns an error.
              */
-            getGrouping(path, page, size, sortString, isAscending, onSuccess, onError) {
-                let endpoint = BASE_URL + "groupings/" + path + "?";
+            getGrouping(groupPaths,
+                        page,
+                        size,
+                        sortString,
+                        isAscending,
+                        onSuccess,
+                        onError) {
+                let endpoint = `${BASE_URL}groupings/group?`;
```

Comment:
This is cool and I like it, but I'm wondering since we don't use this back-tick style in the rest of the file, maybe we should go with the concatenation just for consistency sake. Not sure.

Reply:
It used to be concatenation, but I think Codacy complained about it not being a string literal. I can change it back though.